url-gen/src/test/kotlin/com/cloudinary/transformation/EffectTest.kt

```diff
@@ -156,7 +156,8 @@ class EffectTest {
         cldAssert("fl_waveform", Effect.waveform())
 
         cldAssert(
-            "b_white,co_black,fl_waveform", Effect.waveform {
+            "b_white,co_black,f_png,fl_waveform", Effect.waveform {
```

Comment:
According to spec, it should be `VideoEdit.waveform` and not `Effect.waveform`

Reply:
Checking, thank you